ansible/playbooks/srv/all.yml

```diff
@@ -71,3 +71,10 @@
 - import_playbook: fail2ban.yml
 
 - import_playbook: prosody.yml
+
+- import_playbook: prometheus_exporters.yml
+
+- import_playbook: prometheus_server.yml
+
+- import_playbook: prometheus_alertmanager.yml
+
```

Comment:
Seems this blank line causes test fail in travis.
https://travis-ci.org/debops/debops/jobs/359868266